src/components/homeLeftArea/homeLeftArea.tsx

```diff
@@ -46,28 +46,28 @@ const HomeLeftArea: React.FunctionComponent = () => {
         </Row>
 
         {isSearch
-          ? parentArray?.map((item) => (
-              <Row className={styles.contentStyle} key={item.projectName}>
-                <ProjectContent
-                  heading={item.projectName}
-                  projectDescription={item.description}
-                  uid={item.uid}
-                ></ProjectContent>
-              </Row>
-            ))
-          : []}
+          ? parentArray ?.map((item) => (
```

Comment:
Why space

Reply:
while doing prettier code format the space is coming automatically